aws-datastore/src/test/java/com/amplifyframework/datastore/AWSDataStorePluginTest.java

```diff
@@ -79,8 +87,10 @@ public void setup() {
     @Test
     public void configureAndInitializeInLocalMode() throws AmplifyException {
         //Configure DataStore with an empty config (All defaults)
-        awsDataStorePlugin.configure(new JSONObject(), context);
-        awsDataStorePlugin.initialize(context);
+        ApiCategory emptyApiCategory = spy(ApiCategory.class);
+        AWSDataStorePlugin standAloneDataStorePlugin = new AWSDataStorePlugin(modelProvider, emptyApiCategory);
```

Comment:
You may find utility in the `DataStoreCategoryConfigurator` . Er, maybe thats in `androidTest` though isn't it...